ouroboros-consensus-diffusion/ouroboros-consensus-diffusion.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:   3.0
 name:            ouroboros-consensus-diffusion
-version:         0.8.0.1
+version:         0.9.0.0
```

Comment:
This was just a patch change. Only the patch version should increase, i.e. 0.8.0.2

Reply:
Fixed.